### DISCUSSION TITLE
Can I import data ( transactions ) from yahoo finance?

### DISCUSSION THREAD

**Discussion by bboldi** [ghostfolio/ghostfolio#2242] [Q&A]
Is there a way to import data from Yahoo Finance into Ghostfolio instead of manually entering them one by one? Thanks!

**Comment by dtslvr** [comment]
What format is it? Can you give an example of the transactions exported from _Yahoo Finance_?

**Comment by bboldi** [comment]
Sure:

```
Symbol,Current Price,Date,Time,Change,Open,High,Low,Volume,Trade Date,Purchase Price,Quantity,Commission,High Limit,Low Limit,Comment
IUS2.F,3.703,2023/06/30,15:11 CEST,0.11400008,3.6775,3.719,3.6775,971,20230404,3.77,500.0,,,,
```

**Comment by bboldi** [comment]
I found the example csv and modified it accordingly - managed to import my data, thanks :)

**Reply by dtslvr** [reply]
> I found the example csv and modified it accordingly - managed to import my data, thanks :)

Nice! 👍🏻 

Maybe the importer could be improved by a) extending the [keys](https://github.com/ghostfolio/ghostfolio/blob/main/apps/client/src/app/services/import-activities.service.ts#L17) and b) supporting the [date format](https://github.com/ghostfolio/ghostfolio/blob/main/apps/client/src/app/services/import-activities.service.ts#L212) (`yyyy/MM/dd`). Would you be interested in providing a contribution?

**Reply by bboldi** [reply]
Yeah, why not. I'll see what I can do.

**Reply by dtslvr** [reply]
I have created a ticket: https://github.com/ghostfolio/ghostfolio/issues/2288
